### DISCUSSION TITLE
Virustotal now detect 3 alerts

### DISCUSSION THREAD

**Discussion by pacmankillers** [tranxuanthang/lrcget#146] [General]
Hello, thank you for such an excellent program, it has helped me to keep my collection offline and with lyrics to listen to them or sing them at home. With the intention of being able to help in some way and prevent users from thinking it is a malicious program, I wanted to comment the following hoping for guidance on the matter, as I am not an expert on the subject.

I ran a scan on the latest Windows package, “LRCGET_0.5.0_x64-setup.exe”.

https://www.virustotal.com/gui/file/19001143f21fabd757bf101c1057bb3d0fb1609783319b05bad6265474efc438/detection

While checking it I could notice the following data:

Win/grayware_confidence_60% (D)” detected by CrowdStrike Falcon 2.

Analyzing the information I could notice that it comes from the installer with which it was created, that is to say Nullsoft Scriptable Install System (nsis_tauri_utils.dll), which is what causes this section.

https://www.virustotal.com/gui/file/5ef1f010f9a8be4ffe0913616f6c54acf403ee0b83d994821ae4b6716ec1d266

2. The second detection that I could notice was with “NSIS Ruleset” specifically with the following: “exclude-apis=LdrLoadDll”.

Apparently they are all, from what I have noticed previously the program had been detected falsely but only by an engine in Virustotal, apparently with the latest version has detected more things.

**Comment by alexlambson** [comment]
`nsis_tauri_utils.dll` seems to be a common issue. https://github.com/tauri-apps/nsis-tauri-utils/issues/37
I can say Malware bytes found nothing
